VirtusLab-Open-Source/strapi-plugin-reactions
# Feat/associations sync

## Ticket

https://github.com/VirtusLab-Open-Source/strapi-plugin-reactions/issues/7

## Summary

What does this PR do/solve?

Introduce the **Synchronize associations** admin action and additional special permission to perform it.

## Test Plan

1. Run 
2. User super admin account or add `Settings: Admin` permission to one of your roles
3. Export data
4. Import to new environment with existing data
5. Run the action
6. See all working properly and you're not missing any reactions
